### PR TITLE
Adds black pudding, full english breakfast and a bounty

### PR DESCRIPTION
**Pull request by LodedDiper69**
## About The Pull Request

Adds two new foods to the game: Black Pudding, and the full english breakfast. Also adds a bounty for the breakfast at 6500 credits.

## Why It's Good For The Game

More food is always good ~and it will let us proceed to the next step of the beetannia plannedemic~. The english breakfast has a lot of ingredients and is therefore a nice challenge to try cooking for the bounty.

## Testing Photographs and Procedure
<details>

blackpuddingrecipe

image

britishcuisine
</details>

## Changelog
:cl:
add: Adds black pudding and the full english breakfast
add: Adds a bounty for the aforementioned breakfast
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
I think you borked the food.dmi file a bit

**Comment by PowerfulBacon** [comment]
Yum, good food for once. Hopefully we can see some bangers mash and beans with gravy soon!

File: code/modules/cargo/bounties/chef.dm [hunk 1]
```diff
@@ -136,3 +136,8 @@
 	required_count = 6
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/nugget)
 
+/datum/bounty/item/chef/englishbreakfast
```
Comment (Archanial) [inline code/modules/cargo/bounties/chef.dm]: Please follow the naming scheme

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/cargo/bounties/chef.dm (hunk 1 above)
Comment (LodedDiper69) [reply code/modules/cargo/bounties/chef.dm]: I believe this follows the naming scheme

Comment (LodedDiper69) [reply code/modules/cargo/bounties/chef.dm]: if its about the word "english", then note that we literally have a cuban carp bounty

Comment (Archanial) [reply code/modules/cargo/bounties/chef.dm]: I meant naming it english_breakfast

Comment (LodedDiper69) [reply code/modules/cargo/bounties/chef.dm]: Alright, gonna change that tomorrow once I get some sleep

File: code/modules/cargo/bounties/chef.dm [hunk 2]
```diff
@@ -136,3 +136,8 @@
 	required_count = 6
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/nugget)
 
+/datum/bounty/item/chef/english_breakfast
+	name = "Full English Breakfast"
+	description = "Nanotrasen has received messages about 'QUEEN IN VIRO' from another station. Ship a meal to impress her on her royal visit."
+	reward = 6500
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/englishbreakfast)
```
Comment (itsmeow) [inline code/modules/cargo/bounties/chef.dm]: rename to english_breakfast

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 3]
```diff
@@ -131,6 +131,16 @@
...
 
+/obj/item/reagent_containers/food/snacks/blackpudding
+	name = "black pudding"
+	desc = "Bloody delicious."
+	icon_state = "blackpudding"
+	filling_color = "#5B3330"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/nutriment/vitamin = 2)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/nutriment/vitamin = 2)
+	tastes = list("savoryness" = 1)
+	foodtype = MEAT | BREAKFAST
```
Comment (itsmeow) [inline code/modules/food_and_drinks/food/snacks_meat.dm:142]: if it has meat in it, add the meat taste

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 4]
```diff
@@ -411,6 +411,16 @@
 	tastes = list("cherry" = 1, "crepe" = 1)
 	foodtype = GRAIN | FRUIT | SUGAR
 
+/obj/item/reagent_containers/food/snacks/englishbreakfast
+	name = "full english breakfast"
+	desc = "The maximum amount of breakfasting. Anymore and you reach hubris."
+	icon_state = "englishbreakfast"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/nutriment/vitamin = 1)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/consumable/nutriment/vitamin = 2)
+	tastes = list("a symphony of breakfast foods" = 3)
```
Comment (itsmeow) [inline code/modules/food_and_drinks/food/snacks_other.dm:420]: this should be broken into its constituent parts probably

File: code/modules/food_and_drinks/food/snacks_meat.dm (hunk 3 above)
Comment (itsmeow) [reply code/modules/food_and_drinks/food/snacks_meat.dm:142]: should also have onion taste

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 5]
```diff
@@ -131,6 +131,16 @@
 	. = ..()
 	eatverb = pick("bite","chew","nibble","gobble","chomp")
 
+/obj/item/reagent_containers/food/snacks/blackpudding
```
Comment (itsmeow) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: rename to black_pudding

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 6]
```diff
@@ -411,6 +411,16 @@
 	tastes = list("cherry" = 1, "crepe" = 1)
 	foodtype = GRAIN | FRUIT | SUGAR
 
+/obj/item/reagent_containers/food/snacks/englishbreakfast
```
Comment (itsmeow) [inline code/modules/food_and_drinks/food/snacks_other.dm]: rename to english_breakfast

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 7]
```diff
@@ -333,3 +333,16 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/crab_rangoon
 	subcategory = CAT_MISCFOOD
+
+/datum/crafting_recipe/food/englishbreakfast
```
Comment (itsmeow) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm]: english_breakfast

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm [hunk 8]
```diff
@@ -239,3 +239,13 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/meatclown
 	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/blackpudding
```
Comment (itsmeow) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: black_pudding

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm [hunk 9]
```diff
@@ -239,3 +239,13 @@
...
+	name = "Black Pudding"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/sausage = 1,
+		/obj/item/reagent_containers/food/snacks/onion_slice = 1,
+		/datum/reagent/blood = 10
+	)
+	result = /obj/item/reagent_containers/food/snacks/blackpudding
```
Comment (itsmeow) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: black_pudding

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/food/snacks_meat.dm (hunk 3 above)
Comment (LodedDiper69) [reply code/modules/food_and_drinks/food/snacks_meat.dm:142]: this is already included when you bite a black pudding. savoryness is just an extra taste.

File: code/modules/food_and_drinks/food/snacks_other.dm (hunk 4 above)
Comment (LodedDiper69) [reply code/modules/food_and_drinks/food/snacks_other.dm:420]: see above

File: code/modules/food_and_drinks/food/snacks_meat.dm (hunk 3 above)
Comment (itsmeow) [reply code/modules/food_and_drinks/food/snacks_meat.dm:142]: gotcha

**Review by itsmeow — APPROVED** [review]

**Review by Archanial — DISMISSED** [review]

**Comment by Archanial** [comment]
@PestoVerde322 sprites

**Review by itsmeow — CHANGES_REQUESTED** [review]
Delete the random `food.dmi` in root directory

**Review by itsmeow — CHANGES_REQUESTED** [review]
Review still applies, remove the random `food.dmi` in the root directory

**Comment by Tsar-Salat** [comment]
@LodedDiper69 do you plan on continuing this? 

If not, I can take over for you

**Pull request closed without merging**